Examples/Tests/Macroscopic_Maxwell/inputs_3d.evolveM

```diff
@@ -0,0 +1,88 @@
+geometry.prob_hi     =   32.e-6  32.e-6  512.e-6
+amr.max_level = 0
+
+#################################
+############ NUMERICS ###########
+#################################
+warpx.verbose = 0
+warpx.use_filter = 0
+warpx.cfl = 1.
+warpx.do_pml = 0
+
+#algo.em_solver_medium = vacuum # vacuum/macroscopic
+algo.em_solver_medium = macroscopic # vacuum/macroscopic
+algo.macroscopic_sigma_method = laxwendroff # laxwendroff or backwardeuler
+macroscopic.sigma_init_style = "parse_sigma_function" # parse or "constant"
+macroscopic.sigma_function(x,y,z) = "0.0"
+#macroscopic.sigma = 0.0
+macroscopic.epsilon_init_style = "parse_epsilon_function" # parse or "constant"
+macroscopic.epsilon_function(x,y,z) = "8.8541878128e-12"
+#macroscopic.epsilon = 8.8541878128e-12
+macroscopic.mu_init_style = "parse_mu_function" # parse or "constant"
+macroscopic.mu_function(x,y,z) = "1.25663706212e-06"
+#macroscopic.mu = 1.25663706212e-06
+macroscopic.mag_Ms_init_style = "parse_mag_Ms_function" # parse or "constant"
+macroscopic.mag_Ms_function(x,y,z) = "8e5"
+#macroscopic.mag_Ms = 8e5
+macroscopic.mag_alpha_init_style = "parse_mag_alpha_function" 
+macroscopic.mag_alpha_function(x,y,z) = "1e-04"
+#macroscopic.mag_alpha = 1e-04
+macroscopic.mag_gamma_init_style = "parse_mag_gamma_function"
+macroscopic.mag_gamma_function(x,y,z) = "1.759e-11"
+#macroscopic.mag_gamma = 1.759e-11
+particles.nspecies = 0
+
+#################################
+############ FIELDS #############
+#################################
+
+my_constants.pi = 3.14159265359
+my_constants.L = 141.4213562373095e-6
+my_constants.c = 299792458.
+my_constants.wavelength = 64.e-6
+
+warpx.E_ext_grid_init_style = parse_E_ext_grid_function
+warpx.Ez_external_grid_function(x,y,z) = 0.
+warpx.Ex_external_grid_function(x,y,z) = 0.
+warpx.Ey_external_grid_function(x,y,z) = "1.e5*exp(-z**2/L**2)*cos(2*pi*z/wavelength)"
+
+warpx.B_ext_grid_init_style = parse_B_ext_grid_function
+warpx.Bx_external_grid_function(x,y,z)= "-1.e5*exp(-z**2/L**2)*cos(2*pi*z/wavelength)/c"
+warpx.By_external_grid_function(x,y,z)= 0.
+warpx.Bz_external_grid_function(x,y,z) = 0.
+
+warpx.H_bias_ext_grid_init_style = parse_H_bias_ext_grid_function
+warpx.Hx_bias_external_grid_function(x,y,z)= 50.
+warpx.Hy_bias_external_grid_function(x,y,z)= 0.
+warpx.Hz_bias_external_grid_function(x,y,z)= 0.
+#warpx.H_bias_ext_grid_init_style = constant
+#warpx.Hx_bias_external_grid = 0
+#warpx.Hy_bias_external_grid = 0
+#warpx.Hz_bias_external_grid = 0
+
+#warpx.M_ext_grid_init_style = parse_M_ext_grid_function
+#warpx.Mx_external_grid_function(x,y,z)= "-1.e5*exp(-z**2/L**2)*cos(2*pi*z/wavelength)/c"
+#warpx.My_external_grid_function(x,y,z)= 0.
+#warpx.Mz_external_grid_function(x,y,z) = 0.
+
+warpx.M_ext_grid_init_style = constant
+warpx.M_external_grid = 1. 2. 3.
+
+#warpx.M_ext_grid_init_style = constant
+#warpx.M_external_grid= 1. 2. 3. 1. 2. 3. 1. 2. 3.
+
+
+# Diagnostics
+diagnostics.diags_names = diag1
+diag1.period = 20
+diag1.diag_type = Full
```

Comment:
add:
diag1.fields_to_plot = Ex Ey Ez Bx By Bz Mx_xface Mx_yface Mx_zface My_xface My_yface My_zface Mz_xface Mz_yface Mz_zface

Reply:
I added M plots for now. Thank you for the suggestion.